README.md

```diff
@@ -39,9 +38,9 @@ ran against this extention library. If you are porting your code to this codebas
 Some of the major differences between this library and Numenta's extension library are the following:
 
  * Support for Python 3 and Python 2.7 (Only Python 3 under windows)
- * Support for Linux, OSx, and Windows MS Visual Studio 2017
+ * Cross Platform Support for Linux, OSx, and Windows MS Visual Studio 2017
```

Comment:
- nit, I wouldn't go into details and just say Linux, OSX, MS Windows
- we can now add ARM64 too